core/ui/TagTemplate.tid

```diff
@@ -27,4 +33,4 @@ color:$(foregroundColor)$;
 </span>
 \end
 
-<$macrocall $name="tag-body" colour={{!!color}} palette={{$:/palette}}/>
```

Comment:
Thank you. Will do. Gotta figure out how to delete. Thank you again.


Happy Connecting. Sent from my Sprint Samsung Galaxy S® 5

-------- Original message --------
From: nameanyone <notifications@github.com> 
Date: 5/31/16  1:43 AM  (GMT-07:00) 
To: Jermolene/TiddlyWiki5 <TiddlyWiki5@noreply.github.com> 
Cc: Andrew J Harrison <andrew.j.harrison84@gmail.com>, Mention <mention@noreply.github.com> 
Subject: Re: [Jermolene/TiddlyWiki5] Update TagTemplate.tid (#2469) 

In core/ui/TagTemplate.tid:
> @@ -27,4 +33,4 @@ color:$(foregroundColor)$;
>  </span>
>  \end
>  
> -<$macrocall $name="tag-body" colour={{!!color}} palette={{$:/palette}}/>

No, the newline is still there. Your last two commits are empty. What I did in the past to correct a similar situation was (not sure how detailed you need this to be, ask for more if not clear):


Delete the messed up pull request (this one).
Edit the file locally, upload it to your fork using git push.
Generate a new pull request. Make sure the commit doesn't show the last line as changed. Submit.


HTH.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.